plans/libevent/plan.sh

```diff
@@ -0,0 +1,13 @@
+pkg_origin=core
+pkg_name=libevent
+pkg_version=2.0.22
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
```

Comment:
looks like libevent is licensed under modified BSD: http://libevent.org/LICENSE.txt